src/libraries/Microsoft.XmlSerializer.Generator/tests/Microsoft.XmlSerializer.Generator.Tests.csproj

```diff
@@ -1,8 +1,10 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <DefineConstants>$(DefineConstants);XMLSERIALIZERGENERATORTESTS</DefineConstants>
+    <DisableImplicitFrameworkReferences>false</DisableImplicitFrameworkReferences>
```

Comment:
I think this may be redundant.  I believe tests already get the the implicit reference.